doc/sections/commands.md

```diff
@@ -383,3 +383,13 @@ set builtin "eq"    ≔ eq    // : Π {a}, T a → T a → Prop
 set builtin "refl"  ≔ refl  // : Π {a} (x:T a), P (x=x)
 set builtin "eqind" ≔ eqind // : Π {a} x y, P (x = y) → Π (p:T a→Prop), P (p y) → P (p x)
 ```
+
+**unification rules** The unification engine can be guided using 
+*unification rules*. Given a unification problem `t ≡ u`, if the engine cannot
+find a solution, it will try to match the pattern `t ≡ u` against the defined
+rules and rewrite the problem to the right-hand side of the matched  rule.
+For instance, given the unification rule
+```
+set unif_rule Bool ≡ T $t ↪ $t ≡ bool
```

Comment:
Better provide an example with a comma:  f $x1 $y1 = f $x2 $y2 --> $x1 = $x2, $y1 = $y2

Reply:
You should also add that this is experimental and no sanity check is done, so it may lead to invalid deductions on unification or non-termination.